ios/FluentUI/Button/Button.swift

```diff
@@ -348,49 +268,17 @@ open class Button: UIButton {
         updateBackgroundColor()
         updateBorderColor()
 
-        layer.borderWidth = style.hasBorders ? Constants.borderWidth : 0
+        layer.borderWidth = tokenSet[.borderWidth].float
```

Comment:
feels like it should be inside updateBorderColors() and probably updateBorderColors() function name needs to be updated